src/js/components/Header.js

```diff
@@ -34,13 +34,27 @@ class Header extends Component {
 
           <div className="navbar-links-container">
             <div className="navbar-links">
-              <a className="nav-open" href="/">Get Advice</a>
+              <a className="nav-open" href="/">
+                Get Advice
+              </a>
             </div>
             <div className="navbar-links">
-              <a href="http://www.shinetext.com/referrals">Invite Friends</a>
+              <a
+                className="nav-open"
+                href="/https://www.shinetext.com/app/?utm_source=Shine&utm_medium=Blog&utm_campaign=Top_Nav"
+              >
+                Get The App
+              </a>
             </div>
             <div className="navbar-links">
-              <a href="http://www.shinetext.com/squad">The Squad</a>
+              <a href="http://www.shinetext.com/referrals/?utm_source=Shine&utm_medium=Blog&utm_campaign=Top_Nav">
```

Comment:
And this link and the one below should also direct to `https` instead of `http`.